app/controllers/hiring_staff/base_controller.rb

```diff
@@ -51,4 +52,10 @@ def dsi_logout_url
     url.query = { post_logout_redirect_uri: auth_dfe_signout_url, id_token_hint: session[:id_token] }.to_query
     url.to_s
   end
+
+  def render_link_to_flash_message
```

Comment:
It might be useful also for the jobseekers area, so this could go to the generic ApplicationController

Reply:
I didn't think we would want to, that's why I put it there. But maybe it's nicer to move it there just in case we do want to 👍